bucket_test.go

```diff
@@ -908,6 +908,49 @@ func TestBucket(t *testing.T) {
 						}
 					}
 				})
+
+				if !strings.Contains(impl.name, "S3BucketChecksums") {
```

Comment:
You can add a `t.Skip` with this conditional (instead of a return) in your new testing block below, that should do the trick.

Reply:
So while implementing @johndaniels' later suggestion of verifying the file was modified when the option wasn't used, I ran into the following behavior: If you call bucket.Download(), Pail will respect the `UseSingleFileChecksums` property, but if you use bucket.Pull(), it will not. So for these S3 bucket tests, which use Pull, we should have updated timestamps 100% of the time.

Reply:
Just to clarify: Pull always does checksums, correct?

Reply:
Correct, Pull does checksums 100% of the time regardless of your Bucket settings.